diff --git a/setup.py b/setup.py
index a75b7ed9b..55debad8a 100755
--- a/setup.py
+++ b/setup.py
@@ -1103,6 +1103,7 @@ def build_static_kittens(
     if args.skip_building_kitten:
         print('Skipping building of the kitten binary because of a command line option. Build is incomplete', file=sys.stderr)
         return ''
+    return ''
     cmd = [go, 'build', '-v']
     vcs_rev = args.vcs_rev or get_vcs_rev()
     ld_flags: List[str] = []
