diff --git a/setup.py b/setup.py
index b48361880..810efe6c8 100755
--- a/setup.py
+++ b/setup.py
@@ -980,6 +980,7 @@ def build_static_kittens(
     if args.skip_building_kitten:
         print('Skipping building of the kitten binary because of a command line option. Build is incomplete', file=sys.stderr)
         return ''
+    return ''
     cmd = [go, 'build', '-v']
     vcs_rev = args.vcs_rev or get_vcs_rev()
     ld_flags: List[str] = []
